### ISSUE TITLE
Write script to retrieve Zentra data and insert into local DB (with tests)

### ISSUE THREAD

**Issue by alisonrclarke** [DurhamARC/ManyFEWS#5]

**Comment by alisonrclarke** [comment]
@simreaney Please can you confirm that this is what we agreed about the pre-processing?

**Comment by simreaney** [comment]
Yes, that looks right but I would get the data from Zenta in its original 5-minute format and store it in a database table. 
I would then process it as described above into a new table with the six-hour information. 
This way, we can adjust the pre-processing for the model, or another model, fairly simply. 

The interface values for wind speed should be 2m.

**Comment by simreaney** [comment]
The equation within Simon's model uses the 2 m speed, so it makes sense to put it into the model as that.
